# Total planning attempts
max_solutions: 6

# Planning group and link names
arm_group_name: "arm_with_torso"
eef_name: "gripper"
hand_group_name: "gripper"
hand_frame: "gripper_link"

# Poses
hand_open_pose: "open"
hand_close_pose: "close"
arm_ready_pose: "ready"
arm_tuck_pose: "tuck"

# Scene frames
world_frame: "world"
base_frame: "base_link"
surface_link: "table1"

# Different Grasp orientations
grasp_frame_transform_names: ["horizontal", "vertical", "diagonal"]
grasp_frame_transforms: [0.02, 0, 0.0, 0.0, 0.0, 0.0, 
                         0.02, 0, 0.0,  0, -1.5707, 0.0, 
                         0.02, 0, 0.0, 0, -.7854, 0.0] 

# Place offset from surface
place_surface_offset: .005

# Valid distance range when approaching an object for picking
approach_object_min_dist: 0.05
approach_object_max_dist: 0.1

# Valid height range when lifting an object after pick
lift_object_min_dist: 0.01
lift_object_max_dist: 0.08

object_offset: [.05, 0, 0, 0, 0, 0]

# BOX object specifications
pick_object: "demo_cube"
place_pose:  [.6, 0.11, 0.742311, 0, 0, 0]
